components/collector/src/source_collectors/owasp_zap/security_warnings.py

```diff
@@ -14,32 +14,40 @@ class OWASPZAPSecurityWarnings(XMLFileSourceCollector):
     """Collector to get security warnings from OWASP ZAP."""
 
     async def _parse_entities(self, responses: SourceResponses) -> Entities:
-        """Override to parse the securty warnings from the XML."""
+        """Override to parse the security warnings from the XML."""
         entities: dict[str, Entity] = {}
         tag_re = re.compile(r"<[^>]*>")
         risks = cast(list[str], self._parameter("risks"))
+        count_alert_instances = cast(str, self._parameter("alerts")) == "alert instances"
```

Comment:
I might be misunderstanding this, but does this default to "alert types" right now? Maybe it would be better to default to the old behaviour, so "alert instances".

Reply:
It does default to alert instances. See the default value specified in the data model.

Reply:
ah true